### PR TITLE
Documentation teaser page

### PR DESCRIPTION
**Pull request by anna-lach**
Teaser page for SLDS Documentation
#30 

Possibility to check locally: http://localhost:8080/teaser-page/

After build running needs to be **deployed:**
- docs/website/public/assets/teaser-page
- docs/website/public/assets/fonts
- docs/website/public/teaser-page.html _(our index)_
- docs/website/public/css/teaser-page
- docs/website/public/css/base/base.css
- docs/website/public/css/base/reset.css

### CODE REVIEW

File: docs/website/package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "debug": "DEBUG=* eleventy --input=src",
     "test": "echo \"Error: no test specified\" && exit 1",
     "prettier:write": "prettier --write 'src/**/*'",
-    "lint": "eslint -c .eslintrc.cjs src"
+    "lint:eslint": "eslint -c .eslintrc.cjs src",
```
Comment (jpzwarte) [inline docs/website/package.json:18]: In a future PR, it probably makes sense to start using `wireit` like the rest of the repo.

**Comment by jpzwarte** [comment]
Do we need the `.ttf` files in the repo?

**Comment by jpzwarte** [comment]
Please make the asset filenames (such as the favicon) lowercase, to prevent casing issues.

**Comment by jpzwarte** [comment]
Please rename `github_dark.svg` to `github-dark.svg` like the other files. (just use a consistent file naming convention across all assets plz :) )

File: docs/website/src/styles/base/reset.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* Remove default margin */
+body,
```
Comment (jpzwarte) [inline docs/website/src/styles/base/reset.scss:9]: I read an article recently about using `:where(h1, h2, h3) { margin: 0; }` etc. https://css-tricks.com/using-the-specificity-of-where-as-a-css-reset/

File: docs/website/src/teaser-page.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  
+  <a href="https://sanoma.slack.com/archives/C03SA9HUUA3" class="ds-teaser__card">
+  <div class="ds-teaser__card-text">
+    <p class="ds-teaser__card-text-heading">Slack</p>
+    <span>#sl-design-system</span>
+  </div>
+  <div class="ds-teaser__card-icon ds-teaser__card-icon--slack">
+  </div>
+  </a>
+  </div>
```
Comment (jpzwarte) [inline docs/website/src/teaser-page.md:65]: The indentation of this file is not correct? Or is that just how GitHub displays it?

Comment (anna-lach) [reply docs/website/src/teaser-page.md:65]: Yeah, I fixed it.

**Review by jpzwarte — APPROVED** [review]

**Pull request merged**